resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -206,23 +206,6 @@ func (c *APIClient) CallAPI(path string, method string,
 	return NewAPIResponse(res, body)
 }
 
-func buildURL(basePath string, path string, queryParams map[string]string) (*url.URL, error) {
-	urlString := basePath + path
-	if len(queryParams) > 0 {
-		urlString += "?"
-		for k, v := range queryParams {
-			urlString += fmt.Sprintf("%v=%v&", strings.TrimSpace(k), strings.TrimSpace(v))
-		}
-		urlString = urlString[:len(urlString)-1]
-	}
-
-	u, err := url.Parse(urlString)
-	if err != nil {
-		return nil, err
-	}
-	return u, nil
-}
-
```

Comment:
I noticed this function wasn't being used anymore so i just removed it

Reply:
@dginty4 Have you run a test build on jenkins? I'd do that just to be sure removing this function didn't cause any issues

Reply:
Good idea, thanks Charlie

Reply:
@charliecon it built on Jenkins